match_with_noaa_stations.py

```diff
@@ -149,12 +194,16 @@ def main():
             logging.info(
                 'closest NOAA to %s, is %s, with a distance of %s meters',
                 output['circle_name'],
-                output['closest_station_name'],
+                output['closest_station_id'],
                 output['distance'])
             results.append(output)
     df = pd.DataFrame.from_dict(results)
-    df.to_csv('closest_station.csv.gz', index=False)
+    df.to_csv('closest_stations.csv.gz', index=False)
+    return(df)
+
+# Run the main function to match
+#main(circles_data_smol,noaa_sations)
+closest_stations = main(cbc_usa,noaa_sations)
 
+closest_stations.head()
```

Comment:
Leaving this here will return nothing if you are running this script as a python executable